.github/workflows/quality-checks.yml

```diff
@@ -39,3 +39,6 @@ jobs:
 
     - name: Run linter and static checks
       run: make lint
+
```

Comment:
What platforms do we run this on? If we only run it on one platform can you please create follow up tickets to add test runners for at least linux/amd64 and linux/arm64?

Reply:
Just linux/amd64

[Done](https://viam.atlassian.net/browse/RSDK-7766?atlOrigin=eyJpIjoiNjllZmQ5NTExMWVjNDgwMDk2YWQ3ZmExNzMzYzFkODEiLCJwIjoiaiJ9)